tests/unit/components/drawer/ComponentDetailPanel.spec.js

```diff
@@ -107,6 +109,46 @@ describe('test component: Plugin Component Detail Panel', () => {
     });
   });
 
+  describe('Test function: getUnreferencedAttributes', () => {
+    it('should return an existing attribute', () => {
+      const attributes = { name: 'attribute', definition: null };
+      const component = {
+        attributes: [attributes],
+      };
+
+      expect(wrapper.vm.getUnreferencedAttributes(component)).toEqual([{
+        name: 'attribute',
+        definition: null,
+      }]);
+    });
+  });
+
+  describe('Test function: getSelectedComponentAttributes', () => {
+    it('should return referenced attributes', () => {
+      const refAttributes = { name: 'refAttribute', definition: { definedAttributes: [] } };
```

Comment:
avoid repetition of thoose 3 lines by put them in the `describe` directly above.

Reply:
done